default-templates/server/index.html

```diff
@@ -30,6 +26,41 @@ <h1>Welcome to the Solid Prototype</h1>
 
     If you have not already done so, please create an account.
   </p>
+
+  <p class="lead hidden" id="loggedIn">
+    You are logged in as
+    <a href="#" id="profileLink"></a>.
+  </p>
 </div>
+<script src="/common/js/solid-auth-client.bundle.js"></script>
+<script>
+  const elements = {};
+  ['loggedIn', 'profileLink'].forEach(id => {
```

Comment:
Good support for `Array.prototype.foreach`, but same question as above: https://caniuse.com/#search=forEach

Reply:
That's green everywhere :-)

Reply:
Yeah, but not for older versions of IE...

Reply:
`forEach` is the least of our problems there, many more things are broken. Furthermore, `solid-auth-client` itself (loaded on this page) doesn't support IE: https://github.com/solid/solid-auth-client/blob/v2.2.8/.browserslistrc So it will not work regardless of what syntax I use here.